src/mimir_coordinator.py

```diff
@@ -163,6 +173,14 @@ def _build_compactor_config(self) -> Dict[str, Any]:
             "data_dir": str(self._root_data_dir / "data-compactor"),
         }
 
+    # ring.replication_factor: int
+    # Number of ingesters that each time series is replicated to. This option
+    # needs be set on ingesters, distributors, queriers and rulers when running in
+    # microservices mode.
+    def _build_ingester_config(self) -> Dict[str, Any]:
+        ingester_scale = len(self._cluster_provider.gather_addresses_by_role()[MimirRole.ingester])
+        return {"ring": {"replication_factor": 1 if ingester_scale < 3 else 3}}
```

Comment:
Could we use a constant instead of the magic numbers here, in line 156 and 200?

Reply:
Yes and no, as the `1` will probably stay the same, but I'm changing the expression to:
```python
{"replication_factor": 1 if ingester_scale < REPLICATION_MIN_WORKERS else DEFAULT_REPLICATION}
```

I've also documented those two variables so it's much clearer what they do